automated/dockerfiles/build.sh

```diff
@@ -90,10 +90,36 @@ case "$LTS_SLUG_ARG" in
         ;;
 esac
 
+#
+# Determine if lts slug is latest
+#
+
+SNAPSHOTS="$(mktemp "lts-snapshots.json.XXXXXX")"
+trap "rm -f \"$SNAPSHOTS\"" EXIT
+wget -qO- https://www.stackage.org/download/lts-snapshots.json >"$SNAPSHOTS"
```

Comment:
Is this reintroducing the code that you removed before?  I'm not sure that this is safe, because with the new workflow the snapshot may not yet exist on stackage.org at the point when the image is built.

Reply:
Yes this is the old code with some modification, the logic is that I check whether the minor version of current release is greater or equal to the latest one on stackage.org, to determine whether the current release is the latest version
https://github.com/commercialhaskell/stackage/pull/5265/files#diff-3ad66347364a3a39ec2c2e58f1215adeR159